app/views/admin/people/_people_grid.slim

```diff
@@ -0,0 +1,38 @@
+= define_grid(@people_grid, show_filters: :show, hide_csv_button: true,
+    html: {class: 'rowlink table-hover', data: {link: @csv_mode ? nil : 'row'}}) do |g|
+
+    - g.blank_slate t("no_records")
```

Comment:
Please adjust this to match how I corrected my earlier mistake. See my recent commit to develop.